app/src/main/java/com/ikarabulut/Rules.java

```diff
@@ -0,0 +1,37 @@
+package com.ikarabulut;
+
+public class Rules {
+
+    private static boolean aliveNextGeneration;
+
+    public static boolean aliveCheck(int aliveNeighborsCount) {
```

Comment:
In Java there is a naming convention for methods that return a boolean value.  It is to ask a question in the method using a preposition like `has` or `is`.  The result is that when someone else is reading the code, the method name reads like a question, to which the answer is either "yes" or "no".  It assists code comprehension.

https://stackoverflow.com/questions/3874350/naming-conventions-for-java-methods-that-return-boolean

Reply:
Thank you for sharing this